preview-src/header.tsx

```diff
@@ -97,10 +97,11 @@ const CheckoutButtons = ({ isCurrentlyCheckedOut }) => {
 	}
 };
 
-export function getStatus(state: PullRequestStateEnum) {
+export function getStatus(state: PullRequestStateEnum, isDraft: Boolean) {
```

Comment:
I think this should be `isDraft: boolean`—I think the capitalized version refers to the class boxed type. They may be treated identically, but we use `boolean` elsewhere.